resources/test_fonts/ttx/master_iup.ttx

```diff
@@ -0,0 +1,551 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
IMHO this ttx is complex enough a comment, either in the xml or in something like a sibling master_iup.md, to explain what you are setting up and why it makes for an interesting test would be helpful.

Reply:
Just removed this ttx and reused the vazirmatn_var_trimmed that already exists since it contains an HVAR table.